pkg/apis/messaging/v1alpha1/channel_lifecycle_test.go

```diff
@@ -270,11 +267,16 @@ func TestChannelSetAddressable(t *testing.T) {
 					},
 				},
 				Status: duckv1.Status{
-					Conditions: []apis.Condition{
-						{
-							Type:   ChannelConditionAddressable,
-							Status: corev1.ConditionTrue,
-						}},
+					Conditions: []apis.Condition{{
+						Type:   ChannelConditionAddressable,
+						Status: corev1.ConditionTrue,
+					}, {
+						// Note: Ready is here because when the condition
+						// is marked True, duck automatically sets Ready to
+						// Unknown because of missing ChannelConditionBackingChannelReady.
```

Comment:
Is there a user-visible behavior change here, or was this just a bug in the test?

Reply:
unit tests, and mostly because we don't call InitConditions.

Reply:
Should we call InitConditions in tests?